yarn.lock

```diff
@@ -0,0 +1,9593 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Let's not add this file

Reply:
Sounds good, I removed the file in the PR. Glad to contribute :D